gridfs/grid_file.py

```diff
@@ -761,15 +769,15 @@ def _next_with_retry(self):
         """
         if self._cursor is None:
             self._create_cursor()
-
+        assert self._cursor is not None
```

Comment:
suggest identing this line to avoid duplicating the None check twice in the common case.

Reply:
None